pkg/auth/authz_handler_test.go

```diff
@@ -34,6 +35,11 @@ import (
 	"github.com/thoas/go-funk"
 )
 
+const (
+	FileName  = "test"
+	Directory = "/tmp"
```

Comment:
I'd prefer not to expose test-only constants at the top level.
Especially if they're not named so that it's obvious they're test-related.

I'd say either just don't have consts here and continue using static strings or keep them unexported (lowercase) and name them something test-related (`testAuthDir`, `testAuthFile` or something).

Reply:
Removed consts.